src/SFA.DAS.ApprenticeCommitments.Jobs.UnitTests/WhenApprenticeEmailHasBeenUpdated.cs

```diff
@@ -19,8 +21,8 @@ public class WhenApprenticeEmailHasBeenUpdated
         {
             await sut.Handle(evt, new TestableMessageHandlerContext());
 
-            api.Verify(m => m.UpdateApprenticeEmail(evt.ApprenticeId, It.Is<EmailUpdate>(n =>
-                n.Email == evt.NewEmailAddress)));
+            api.Verify(m => m.UpdateApprentice(evt.ApprenticeId, It.Is<JsonPatchDocument<Api.Apprentice>>(n =>
+                n.Operations.Count == 1 && n.Operations[0].OperationType == OperationType.Replace && n.Operations[0].path == "/Email" && (string)n.Operations[0].value == evt.NewEmailAddress)));
```

Comment:
That's a bit ugly isn't it!  There's not an obviously better way to do it.

Reply:
I agree, we could wrap it up in a verify function, which would make it easier to read